### PR TITLE
wasm2wat: add page

### PR DESCRIPTION
**Pull request by agnivade**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/common/wasm2wat.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# wasm2wat
+
+> Convert a file from the WebAssembly binary format to the text format.
```
Comment (mfrw) [inline pages/common/wasm2wat.md]: Perhaps `WebAssembly binary format to text format` would read better.
Drop the `the`.
For line 3, 5

Comment (sbrl) [reply pages/common/wasm2wat.md]: Not sure on that, @mfrw. The `the` is important in making the sentence read naturally, IMHO.

Comment (mfrw) [reply pages/common/wasm2wat.md]: @sbrl I have to improve my english :)

Comment (sbrl) [reply pages/common/wasm2wat.md]: Practice makes perfect :-)

Comment (agnivade) [reply pages/common/wasm2wat.md]: :)

**Pull request merged**